src/current/v24.1/work-with-virtual-clusters.md

```diff
@@ -114,7 +114,7 @@ sql_txn_commit_count{tenant="demo"} 0
 
 When connected to a virtual cluster from the DB Console:
 
-- Most pages and views are scoped to a virtual cluster. By default the DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system virtual cluster. To allow the DB Console to display system-level metrics from within a virtual cluster, you can grant the virtual cluster the `can_view_node_info` permission.
+- Most pages and views are scoped to a virtual cluster. The DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system virtual cluster.
```

Comment:
```suggestion
- Metrics which measure SQL and related activity show data scoped to the virtual cluster. All other metrics are collected system-wide and display the same data on all virtual clusters including the system virtual cluster.
```

Reply:
I made the two sections identical.

Reply:
Actually, because the heading here is `### Work with metrics with cluster virtualization enabled` I edited to **only** address the bullet point re: metrics in DB Console and not other DB console things since they did not seem to belong.